docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -201,6 +201,15 @@ a|The command line (CLI) arguments to use when starting browser.
 
 |===
 
+NOTE: For using Opera browser it is mandatory property:
```

Comment:
Is this the right place for this? Maybe instead add a description for the web/desktop/opera profile and include this in the description

Reply:
moved to `Profiles`